spec/scheme_builder_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "spec_helper"
+
+describe Liftoff::SchemeBuilder do
+  describe "#create_schemes" do
+    it "creates schemes with the right names" do
+      project = double("xcodeproj")
+      config = build_config("FakeApp")
+      scheme_builder = scheme_builder(project, config)
+
+      expect(project).to receive(:generate_default_scheme).with(no_args)
+      expect(project).to receive(:generate_scheme).with("FakeApp-CI")
+      expect(project).to receive(:generate_scheme).with("FakeApp-AppStore")
+
+      scheme_builder.create_schemes
+    end
+
+    it "add build configurations to actions" do
+      project = double("xcodeproj")
+      schemes = [build_scheme("<%= project_name %>-CI")]
+      config = build_config("FakeApp", schemes)
+      scheme_builder = scheme_builder(project, config)
+
+      scheme = Xcodeproj::XCScheme.new
+
+      expect(project).to receive(:generate_default_scheme).with(no_args)
+      expect(project).to receive(:generate_scheme).with("FakeApp-CI").and_yield(scheme)
+      
+      scheme_builder.create_schemes
+
+      expect(scheme.test_action.build_configuration).to eq "Debug-CI"
+      expect(scheme.profile_action.build_configuration).to eq "Release-CI"
+      expect(scheme.analyze_action.build_configuration).to eq "Debug-CI"
+      expect(scheme.archive_action.build_configuration).to eq "Release-CI"
+      expect(scheme.launch_action.build_configuration).to eq "Debug-CI"
+    end
+
+  end
+
+  def scheme_builder(project, config)
+    Liftoff::SchemeBuilder.new(project, config)
+  end
+
+  def build_config(name, schemes=build_schemes())
```

Comment:
Thoughts on leaving off the default value here and passing it in where you left it out in LN8?